notebooks/auxtel/observing_runs/20230310/latissWepPipeline.yaml

```diff
@@ -0,0 +1,56 @@
+# This yaml file is used to define the tasks and configuration of
+# a Gen 3 pipeline used for testing in ts_wep.
+description: wep basic processing test pipeline
+instrument: lsst.obs.lsst.Latiss
+# Use imported instrument configuration
+imports:
+  - location: $TS_WEP_DIR/policy/cwfs/instData/auxTel/instParamPipeConfig.yaml
+tasks:
+  isr:
+    class: lsst.ip.isr.isrTask.IsrTask
+    config:
+      connections.outputExposure: postISRCCD
+      doApplyGains: false
+      doBias: true
+      doBrighterFatter: false
+      doCrosstalk: false
+      doDark: true
+      doDefect: true
+      doFlat: true
+      doFringe: true
+      doInterpolate: true
+      doLinearize: false
+      doNanMasking: true
+      doOverscan: true
+      doVariance: false
+      python: OverscanCorrectionTask.ConfigClass.fitType = 'MEDIAN_PER_ROW'
+  generateDonutDirectDetectTask:
+    class: lsst.ts.wep.task.GenerateDonutDirectDetectTask.GenerateDonutDirectDetectTask
+    config:
+      # connections.exposure: ic_exp
+      # Obscuration (inner_radius / outer_radius of primary mirror)
+      instObscuration: 0.3525
+      # Focal length in m
+      instFocalLength: 21.6
+      # Aperture diameter in m
+      instApertureDiameter: 1.2
+      # Move the M2 to fake the move of focal plane dz
+      # Move the M2 by 0.8 mm = move the focal plane by 32.8 mm  (magnification of 41)
+      # Defocal distance offset in mm
+      instDefocalOffset: 32.8
+      # Camera pixel size in m
+      instPixelSize: 10.0e-6
+      opticalModel: 'onAxis'
+      donutSelector.useCustomMagLimit: True
+      # Pick the brightest source only
+      donutSelector.sourceLimit: 1
+  cutOutDonutsScienceSensorTask:
+    class: lsst.ts.wep.task.CutOutDonutsScienceSensorTask.CutOutDonutsScienceSensorTask
+    config:
+      # And here we specify the configuration settings
+      # connections.exposure: ic_exp
+      donutTemplateSize: 200
+      donutStampSize: 200
+      initialCutoutPadding: 40
+  calcZernikesTask:
+    class: lsst.ts.wep.task.CalcZernikesTask.CalcZernikesTask
```

Comment:
I think this is still missing `transposeImages: False`

Reply:
Forgot to `git push`. *Facepalm*